src/DefaultBuilder/src/GenericHostBuilderExtensions.cs

```diff
@@ -25,6 +25,11 @@ public static class GenericHostBuilderExtensions
         /// <returns>The <see cref="IHostBuilder"/> for chaining.</returns>
         public static IHostBuilder ConfigureWebHostDefaults(this IHostBuilder builder, Action<IWebHostBuilder> configure)
         {
+            if (configure is null)
```

Comment:
Should I bother with this? This would have null-ref (so would have using a null `IHostBuilder`).